Services/OpenIdConnect/classes/class.ilOpenIdConnectSettingsGUI.php

```diff
@@ -27,7 +27,9 @@ class ilOpenIdConnectSettingsGUI
     private const STAB_ROLES = 'roles';
 
     private const DEFAULT_CMD = 'settings';
-    private int $ref_id = 0;
+    private int $ref_id;
+    /** @var mixed | null | object */
```

Comment:
@pascalseeland IMO this is always an array `$DIC->http()->request()->getParsedBody();`